2-mandatory/2-financial-times.js

```diff
@@ -14,68 +24,108 @@ function potentialHeadlines(allArticleTitles) {
     (you can assume words will always be seperated by a space)
 */
 function titleWithFewestWords(allArticleTitles) {
-    // TODO
+  // TODO
+
+  headlines = potentialHeadlines(allArticleTitles);
+
+  let shortHeadline = [];
+  let fewestWords = 100;
```

Comment:
This is not bad - and it might work for the test cases...
But what if all the headlines had more that 100 words?
Can you think of how you could do this without initialising `fewestWords` to a specific number?
One idea is to use the number of words in the first headline 🤔